keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -683,13 +696,29 @@ rollout. Similarly, consider large clusters and how enablement/disablement
 will rollout across nodes.
 -->
 
+If a control plane rollout disables this feature, the StatefulSet controller
+will update ordinal numbers it controls. This will result in pods being deleted,
+while other pods are scaled in. The StatefulSet controller scales up pods before
+it deletes pods, so as a result, the StatefulSet should not manage fewer than the
+number of replicas that are defined in the spec. Disabling the feature may have an effect on the
+stateful workload that is being run. If the stateful application expects a
+specific ordinal number to be available, it may result in an application failing
+to reach quorum, or rebalancing data based on the number of available replicas.
+
 ###### What specific metrics should inform a rollback?
 
 <!--
 What signals should users be paying attention to when the feature is young
 that might indicate a serious problem?
 -->
 
+The `kube_statefulset_status_replicas` metric can be monitored against the
+`kube_statefulset_replicas` metric to check the expected number of replicas to
```

Comment:
Are those kube-state metrics? Are those per stateful-set? if so how does it handle cardinatlity issues if I have gazillion of stateful sets?

Reply:
Yes, these are kube-state-metrics. Cardinality issues can only be checked on a fleet level if it's expected a certain number of StatefulSets will fall into the steady state category. Added some commentary on how this might be checked on a larger scale.